packages/storage/src/reference.ts

```diff
@@ -52,12 +53,15 @@ import { UploadResult } from './tasksnapshot';
  *     format. If no value is passed, the storage object will use a URL based on
  *     the project ID of the base firebase.App instance.
  */
-export class StorageReference {
+export class Reference {
   /**
    * @internal
    */
   _location: Location;
 
+  /**
+   * @internal
```

Comment:
Do we need any of these annotations for all implementation classes? I think we don't since our public API references only types instead of these concrete classes.

Reply:
Removed all the `@internal` tags that are not needed anymore after switching to public types.